src/tile.cpp

```diff
@@ -359,35 +356,36 @@ void Tile::onAddTileItem(Item* item)
 
 	setTileFlags(item);
 
-	const Position& cylinderMapPos = getPosition();
+	const Position& thingMapPos = getPosition();
 
 	SpectatorVec spectators;
-	g_game.map.getSpectators(spectators, cylinderMapPos, true);
+	g_game.map.getSpectators(spectators, thingMapPos, true);
 
 	// send to client
-	for (Creature* spectator : spectators) {
-		if (Player* spectatorPlayer = spectator->getPlayer()) {
-			spectatorPlayer->sendAddTileItem(this, cylinderMapPos, item);
+	for (const auto& spectator : spectators) {
+		if (auto spectatorPlayer = spectator->getPlayer()) {
```

Comment:
Could this be `const auto`
```suggestion
		if (const auto spectatorPlayer = spectator->getPlayer()) {
```